experimental/GModule/GModule.jl

```diff
@@ -191,10 +196,27 @@ where `M` is a module over `R`.
 
 (modules over finite fields or number fields)
 """
-function descent_to_minimal_degree_field(M::GModule)
-  error("not yet ...")
+function descent_to_minimal_degree_field(C::GModule{<:Any, <:AbstractAlgebra.FPModule{fpFieldElem}})
+  return C
 end
 
+function descent_to_minimal_degree_field(C::GModule{<:Any, <:AbstractAlgebra.FPModule{<:FinFieldElem}})
+  #always over char field
+  K =  base_ring(C)
+  d = 0
+  while d < degree(K)-1
+    d += 1
+    degree(K) % d == 0 || continue
+    k = GF(Int(characteristic(K)), d)
```

Comment:
What if the characteristic of `K` does not fit into an `Int`?

Actually with the new finite fields, I think it doesn't matter anyway, you can just do
```suggestion
    k = GF(characteristic(K), d)
```
and it will decide internally which representation to use... Right, @thofma ?

Reply:
I agree, this is just automatic if we do it this way.

Reply:
Yes. We don't need these shenanigans anymore.